info/Joe Johnson.html

```diff
@@ -0,0 +1,21 @@
+<style>
+	body {border-top: thick double black;}
+</style>
+<html>
+	<body>
+		<nav>
+			<center><i><a href="www.google.com">About Me</a></i> | <a href="www.google.com">Portfolio</a></center>
+		</nav>
+		<center><h1>Joe Johnson</h1></center>
```

Comment:
The center tag is not supported in HTML5 so do text alignment in CSS instead: http://www.w3schools.com/css/css_text.asp